packages/documentation/copy/en/get-started/TS for the New Programmer.md

```diff
@@ -45,7 +45,7 @@ Every language has its own _quirks_ — oddities and surprises, and JavaScript's
   ```js
   const obj = { width: 10, height: 15 };
   // Why is this NaN? Spelling is hard!
-  const area = obj.width * obj.heigth;
+  const area = obj.width * obj.height;
```

Comment:
per the comment above, these are _intentional_ spelling errors to demonstrate an error